docs/content/s3.md

```diff
@@ -647,44 +647,19 @@ To configure access to IBM COS S3, follow the steps below:
 	storage_class>
 ```
 
-12. Review the displayed configuration and accept to save the "remote" then quit.
+12. Review the displayed configuration and accept to save the "remote" then quit. The config file should look like this
 ```
-	Remote config
-	--------------------
-	[IBM-COS-XREGION]
 	env_auth = false
-	access_key_id = <>
+	access_key_id = msRoDENqLymO37eaOA1s
```

Comment:
This looks like an actual Access key - did you mean to leave this here?

Reply:
oops.. did not mean to include that.. should i resubmit?